TCMSApp/src/client/app/blocks/fakers/fake-tests.factory.js

```diff
@@ -102,7 +102,7 @@
                     var testCase = {};
 
                     testCase._id = faker.random.uuid();
-                    testCase.testName = faker.lorem.sentence(2, 7);
+                    testCase.testName = faker.lorem.sentence(2, 3);
```

Comment:
2, 3 means that we will get the sentence from 2 to 3 words? I think that 2-7 is more reasonable